### PR TITLE
update t-digest algo in tex

### PR DESCRIPTION
**Pull request by CamDavidsonPilon**
@tdunning, line `62` was confusing me - can you confirm it should be `j` instead of `i`? Also, can you confirm the other changes as well?

### CODE REVIEW

**Comment by tdunning** [comment]
Can you justify the changes?  Looks like you are correcting typos and such,
but it would help to have some commentary.

**Comment by CamDavidsonPilon** [comment]
Yes, they are just typo corrections, no new logic. I'll comment inline.

File: docs/t-digest-paper/histo.tex [hunk 1]
```diff
@@ -56,10 +56,10 @@ \section{The Algorithm}
 \KwOut{final set $C=[c_1 \ldots c_m]$ of weighted centroids} 
 \For{$(x_n, w_n) \in X$} {
   $z = \min | c_i.\mathtt{mean} - x |$\;
-  $S = \lbrace c_i  :  |c_i.\mathtt{mean} - x| = z \wedge c_i.\mathtt{count} + 1 \le 4n\delta q(c_i) (1-q(c_i)) \rbrace $\;
+  $S = \lbrace c_i  :  |c_i.\mathtt{mean} - x| = z \wedge c_i.\mathtt{count} + w_n \le 4n\delta q(c_i) (1-q(c_i)) \rbrace $\;
```
Comment (CamDavidsonPilon) [inline docs/t-digest-paper/histo.tex]: We need to exclude Centroids that have less than the necessary space left. `1` is fine when the data has weight `1`, but in general (like the algorithm here states), we assume a weight of `w_n`.

File: docs/t-digest-paper/histo.tex [hunk 2]
```diff
@@ -56,10 +56,10 @@ \section{The Algorithm}
...
    \While {$S \ne \lbrace \rbrace\wedge w_n > 0$} {
        Sample $c_j \sim \mathrm{Uniform}( S)$\;
-       $\Delta w = \min(4n\delta q(c_i) (1-q(c_i))-c_j.\mathtt{count}, w_n)$\;
+       $\Delta w = \min(4n\delta q(c_j) (1-q(c_j))-c_j.\mathtt{count}, w_n)$\;
```
Comment (CamDavidsonPilon) [inline docs/t-digest-paper/histo.tex]: There is no `c_i`, so I'm assuming it must be `c_j`.

File: docs/t-digest-paper/histo.tex [hunk 3]
```diff
@@ -131,7 +131,7 @@ \subsection{Finding the cumulative distribution at a point}
     }
     $z = \max(-1, (x-m_i)/\Delta)$\;
   \If {$z < 1$} {
-    \Return $(t + \frac{k_i} N \frac{z+1} {2})$
+    \Return $( \frac{t}{N} + \frac{k_i} N \frac{z+1} {2})$
```
Comment (CamDavidsonPilon) [inline docs/t-digest-paper/histo.tex]: probably just a typo.

Comment (tdunning) [inline docs/t-digest-paper/histo.tex]: Great.  Thanks.  Makes total sense.

**Pull request merged**